transforms/code/license_select/python/README.md

```diff
@@ -0,0 +1,92 @@
+# License Select 
+
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+
+The License Select transform checks if the `license` of input data is in approved/denied list. It is implemented as per the set of [transform project conventions](../../README.md#transform-project-conventions) the following runtimes are available:
+This filter scans the license column of an input dataset and appends the `license_status` column to the dataset.
+
+The type of the license column can be either string or list of strings. For strings, the license name is checked against the list of approved licenses. For list of strings, each license name in the list is checked against the list of approved licenses, and all must be approved.
+
+If the license is approved, the license_status column contains True; otherwise False. 
+
+## Configuration and command line Options
+
+The set of dictionary keys holding license_select configuration for values are as follows:
+
+The transform can be configured with the following key/value pairs from the configuration dictionary.
+
+```python
+# Sample params dictionary passed to the transform
+
+{ 
+"license_select_params" : {
+        "license_column_name": "license",
+        "deny_licenses": False,
+        "licenses": [ 'MIT', 'Apache'],
+        "allow_no_license": False,
+    }
+}
+```
+license_column_name - The name of the column with licenses.
```

Comment:
this should be a list of bullets. as is, it is wrapping into a single paragraph.

Reply:
done